<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserisciNuovoAudiolibroDialog</class>
 <widget class="QDialog" name="InserisciNuovoAudiolibroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibreriaDigitale</string>
  </property>
  <widget class="QLabel" name="autoreAULabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Autore libro:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="autoreAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="AUInseritoLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>220</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Il seguente audiolibro Ã¨ stato correttamente aggiunto alla libreria:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="casaEditriceAULabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Casa editrice:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inserisciNuovoAUButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius: 8px;
background-color:#333333; 
border: none;
color: white;
padding: 2px;
text-align: center;
font-size: 15px;
font-family: &quot;Garamond&quot;;
}

:hover {
background-color: #676767;
}

:pressed{
border: 1px solid #E0E2E2;
}</string>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="dataAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="casaEditriceAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="titoloAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="casaDiscograficaAULabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Casa discografica:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="casaDiscograficaAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 10pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="dataAULabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Data:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="auInseritoTextEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 10pt;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titoloAULabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Titolo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="durataAULabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Durata:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="durataAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="autoreAudAULabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Autore traccia vocale:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="autoreAudAUTextEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
